### ISSUE TITLE
Bug: trained-physics β₁=0 zeroes out decode compute, ~2x throughput overestimate at high batch sizes

### ISSUE THREAD

**Issue by namasl** [inference-sim/inference-sim#1123]
**Describe the bug**

The trained-physics latency model's decode compute correction coefficient β₁ is zero, causing decode step time to depend only on KV memory bandwidth. This is well-calibrated at low concurrency (batch=1 is memory-bandwidth-bound), but underestimates step time at high batch sizes where decode compute becomes significant. The result is ~2x throughput overestimate at c=1024 for non-preemption workloads.

The roofline model does NOT have this issue — it accumulates per-request FLOPs and properly saturates via `max(compute, memory)`.

| Workload (short-short) | Real tok/s | Roofline tok/s | Trained-physics tok/s |
|------------------------|-----------|---------------|----------------------|
| c=1 | 155 | 328 | 143 |
| c=64 | 7,296 | 12,784 | 7,569 |
| c=256 | 15,964 | 17,410 | 20,670 |
| c=1024 | 16,471 | **17,414** ✓ | **35,525** ✗ (2.15x) |

**Root cause**

In `defaults.yaml` (line 1667), the 10-beta trained-physics coefficients are:

```
beta_coeffs: [0.152128, 0.0, 1.36252915, 0.752037, 32.09546717, 4.41684444, 126.024825, 481.8613888, 0.0, 1.94710771]
```

β₁ (index 1, decode compute correction) = **0.0**. β₉ (index 9, decode KV bandwidth) = 1.95.

In 10-beta split mode (`sim/latency/trained_physics_model.go:300-301`):

```go
decodeTerm = m.Beta[1]*tDcCompute + m.Beta[9]*tDcKv
           = 0.0 * tDcCompute + 1.95 * tDcKv
```

The decode compute basis functions (`flopsProj`, `flopsAttn`, `flopsFfn` — lines 221-232) are computed but then multiplied by zero. Step time is dominated by `tWeight` (constant weight loading, ~7ms for Llama-8B on H100) plus the KV bandwidth term, which grows slowly with batch size. Throughput scales nearly linearly with concurrency instead of saturating when compute should become the bottleneck.

**CLI command to reproduce**

```bash
# Trained-physics (2.15x over at c=1024):
./blis run --model meta-llama/Llama-3.1-8B-Instruct \
  --hardware H100 --tp 1 --num-instances 1 \
  --latency-model trained-physics \
  --concurrency 1024 --num-requests 20480 \
  --prompt-tokens 128 --prompt-tokens-stdev 0 --prompt-tokens-min 1 --prompt-tokens-max 128 \
  --output-tokens 128 --output-tokens-stdev 0 --output-tokens-min 1 --output-tokens-max 128 \
  --max-num-running-reqs 1024 --max-num-scheduled-tokens 2048 \
  --long-prefill-token-threshold 2048 --seed 42
# tokens_per_sec: 35524 (expected ~16000)

# Roofline (correct saturation):
./blis run --model meta-llama/Llama-3.1-8B-Instruct \
  --hardware H100 --tp 1 --num-instances 1 \
  --latency-model roofline \
  --concurrency 1024 --num-requests 20480 \
  --prompt-tokens 128 --prompt-tokens-stdev 0 --prompt-tokens-min 1 --prompt-tokens-max 128 \
  --output-tokens 128 --output-tokens-stdev 0 --output-tokens-min 1 --output-tokens-max 128 \
  --max-num-running-reqs 1024 --max-num-scheduled-tokens 2048 \
  --long-prefill-token-threshold 2048 --seed 42
# tokens_per_sec: 17414 (close to real 16471)
```

**Expected behavior**

Trained-physics throughput should saturate at high concurrency, matching roofline (~17.4K tok/s) and real vLLM (~16.5K tok/s).

**Actual behavior**

```
tokens_per_sec: 35524   (2.15x over real)
itl_mean_ms:    27.8    (should be ~62ms to match real throughput)
preemption_count: 0     (clean test — no preemption confounds)
```

**Which invariant is violated?**
- [x] None / unknown

This is a coefficient calibration issue, not an invariant violation.

**Environment**
- OS: Linux (Fedora 42)
- Go version: 1.22+
- Commit hash: ca0df8f (main)

**Additional context**

The iter29 training (noted in `defaults.yaml` line 1649) likely optimized β₁ to zero because the training data was dominated by small batch sizes where decode is genuinely memory-bandwidth-bound and the KV term alone provides a good fit. The coefficient needs re-training with data at high batch sizes (128–1024).

Potential fixes:
- Retrain β₁ against real vLLM measurements at batch sizes 128, 256, 512, 1024
- Constrain β₁ > ε during optimization so decode compute is never fully zeroed out
- Use the roofline model as a floor: `decodeTerm = max(roofline_decode, trained_physics_decode)`

Split from #1115. See also #1121 (preemption prefix caching blowup) and #1122 (E2E metric corruption).

**Comment by namasl** [comment]
**Update:** The 45x throughput blowup that prompted this investigation was caused by the 300s default timeout (#1127), not the β₁=0 calibration gap. With timeout disabled, long-long and short-long workloads produce correct throughput at all concurrency levels for both roofline and trained-physics.

This issue remains valid — the β₁=0 coefficient causes trained-physics to overestimate short-short throughput by ~2x at c=1024 (35,525 vs roofline 17,414 vs real 16,471). The roofline model does not have this issue and properly saturates. The gap is specific to workloads where decode compute (rather than KV bandwidth) is the bottleneck at large batch sizes.
